app/views/admin/resources/_form.html.haml

```diff
@@ -0,0 +1,16 @@
+.row
+  .col-md-12
+    .page-header
+      %h1
+        - if @resource.new_record?
+          New
+        = @resource.name
+        Resource
+.row
+  .col-md-8
+    = semantic_form_for(@resource, :url => (@resource.new_record? ? admin_conference_resources_path : admin_conference_resource_path(@conference.short_title, @resource))) do |f|
```

Comment:
used should also be in this form as you could also want to edit it while being in the edit page. :slightly_smiling_face: